crates/cli/src/cli/mod.rs

```diff
@@ -223,22 +225,47 @@ impl StartSimnet {
             }
         }
 
-        for keypair_path in self.airdrop_keypair_path.iter() {
+        let mut airdrop_keypair_path = self.airdrop_keypair_path.clone();
+        if airdrop_keypair_path.is_empty() {
+            using_default_keypair = true;
+            airdrop_keypair_path.push(DEFAULT_SOLANA_KEYPAIR_PATH.clone());
+        }
+
+        let default_resolved_path = resolve_path(DEFAULT_SOLANA_KEYPAIR_PATH.as_str());
```

Comment:
Rather having multiple bool checks (`using_default_keypair` and setting the `default_resolved_path` and checking for equality later), can we just:
```rust
if airdrop_keypair_path.is_empty() {
   // if default path exists use it
   // if not just return 
}
else {
  // use provided values
}

```